_includes/homepage/homepage-usedby-band.html

```diff
@@ -4,10 +4,11 @@
       <h3 class="text-centered blue">Strimzi is used by</h3>
     </div>
 
-    <div class="width-3-12 width-12-12-m logo-img"><img src="{{site.baseurl}}/assets/images/helvetia_logo.png"></div>
-    <div class="width-3-12 width-12-12-m logo-img"><img src="{{site.baseurl}}/assets/images/lightbend_logo.png"></div>
-    <div class="width-3-12 width-12-12-m logo-img"><img src="{{site.baseurl}}/assets/images/redhat_logo.png"></div>
-    <div class="width-3-12 width-12-12-m logo-img"><img src="{{site.baseurl}}/assets/images/axual_logo.png"></div>
+    <div class="width-4-12 width-12-12-m logo-img"><img src="{{site.baseurl}}/assets/images/helvetia_logo.png"></div>
+    <div class="width-4-12 width-12-12-m logo-img"><img src="{{site.baseurl}}/assets/images/lightbend_logo.png"></div>
+    <div class="width-4-12 width-12-12-m logo-img"><img src="{{site.baseurl}}/assets/images/redhat_logo.png"></div>
+    <div class="width-4-12 width-12-12-m logo-img"><img src="{{site.baseurl}}/assets/images/axual_logo.png"></div>
+    <div class="width-4-12 width-12-12-m logo-img"><img src="{{site.baseurl}}/assets/images/grupomasmovil_logo.png"></div>
```

Comment:
I think you should try to center it. Maybe this can help:

```suggestion
    <div class="width-2-12 width-12-12-m"></div>
    <div class="width-4-12 width-12-12-m logo-img"><img src="{{site.baseurl}}/assets/images/axual_logo.png"></div>
    <div class="width-4-12 width-12-12-m logo-img"><img src="{{site.baseurl}}/assets/images/grupomasmovil_logo.png"></div>
    <div class="width-2-12 width-12-12-m"></div>
```

Reply:
I see them already centered